src/tests/ftest/util/telemetry_utils.py

```diff
@@ -476,12 +477,8 @@ def get_all_server_metrics_names(self, server, with_pools=False):
             all_metrics_names.extend(self.ENGINE_POOL_METRICS)
             all_metrics_names.extend(self.ENGINE_CONTAINER_METRICS)
 
-        # Add engine network metrics for the configured provider
-        try:
-            provider = re.sub("[+;]", "_", server.manager.job.get_config_value("provider"))
-        except TypeError:
-            provider = "ofi_sockets"
-        net_metrics = [name.replace("<provider>", provider) for name in self.ENGINE_NET_METRICS]
+        # Add engine network metrics for the native provider
+        net_metrics = [name.replace("<provider>", "native") for name in self.ENGINE_NET_METRICS]
```

Comment:
Does this mean that all the `engine_net_<provider>_*` metric names will now be `engine_net_native_*` with no plans to use something different than `native`?  If so we could just remove this line and update the ENGINE_NET_METRICS definition with `<provider>` replaced by `native`.

Reply:
There will be a secondary and potential more once we have multi-provider support. That's why i left it as is for now.